### DISCUSSION TITLE
Please help to correctly change the payment address to the farmer and the payment address to the pool!

### DISCUSSION THREAD

**Discussion by AleksandrPDA** [Chia-Network/chia-blockchain#9068] [Q&A]
Hello! Please tell me how to correctly change the farmer's payout address and the pool payout address on page 3? Do you need to replace them with the address for receiving payment from page 1 or with the payment address that is on page 4 in the pool?
Thank you in advance!
Привет! Подскажите, пожалуйста, как правильно изменить адрес выплаты фермера и адрес выплаты пула на странице 3? Нужно ли заменять их адресом для получения платежа со страницы 1 или адресом платежа, который находится на странице 4 в пуле?
Заранее спасибо!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
